src/main/java/model/EventModel.java

```diff
@@ -110,23 +107,28 @@ public boolean deleteEvent(String id) {
 		return result;
 	}
 	
-	public boolean resolveParticipant(Map<String, String[]> parameters) {
+	public boolean resolveParticipant(ParticipantBean participant) {
 		boolean result = false;
-		if(parameters != null && !parameters.isEmpty() && parameters.containsKey("event") && parameters.containsKey("user") && parameters.containsKey("action")) {
-			if(checkInteger(parameters.get("event")[0]) && checkInteger(parameters.get("user")[0]) && checkInteger(parameters.get("action")[0])) {
-				int eventId = Integer.parseInt(parameters.get("event")[0]);
-				int userId = Integer.parseInt(parameters.get("user")[0]);
-				int action = Integer.parseInt(parameters.get("action")[0]);
-				switch (action) {
-					case 1: result = daoFactory.getEventDao().enterEvent(eventId, userId); break;
-					case 2: result = daoFactory.getEventDao().leaveEvent(eventId, userId); break;
-					case 3: result = daoFactory.getEventDao().sendRequest(eventId, userId); break;
-					case 4: result = daoFactory.getEventDao().cancelRequest(eventId, userId); break;
-					case 5: result = daoFactory.getEventDao().approveUser(eventId, userId); break;
-					case 6: result = daoFactory.getEventDao().blockUser(eventId, userId); break;
-					case 7: result = daoFactory.getEventDao().removeUser(eventId, userId); break;
-					default: return result;
-				}
+		if(participant.getEventId() != 0 && participant.getUserId() != 0) {
+			int eventId = participant.getEventId();
+			int userId = participant.getUserId();
+			switch (ActionEnum.fromInt(participant.getAction())) {
+				case ENTER_EVENT:
+					result = daoFactory.getEventDao().joinEvent(eventId, userId, 1); 
```

Comment:
I believe we should avoid using hardcoded values, when someone new to the code sees a method call such as joinEvent(eventId, userId, 1) he instantly knows that a user will be joined to an event and he can clearly see that there is an event id and user id used by this method but 1 means nothing, It is either unnecessary or it could be replaced with enum, if it is replaced with enum you should also look into the method itself and if that method will ever get a different value instead of 1 or is this the only place it is called, if there are no other calls you can remove the third parameter and set that value from an enum directly in that method. Same applies to the rest of the methods in this switch.